scripts/publish-go.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 #
-# A script to generate language specific code from source protocol files and
-# publish them to downstream GitHub repositories.
+# A script to publish generate language specific code to downstream GitHub repositories.
```

Comment:
This sentence now doesn't seem to make sense, should it perhaps be:

"A script to publish generated code to downstream GitHub repositories."

Reply:
Thanks. Done: https://github.com/ably/ably-common/pull/90/commits/6bdafd8a7dbc9ae950517cfbc45ac1f562d522e3

I'm going to take the liberty of resolving this conversation on your behalf so I can get this pull request landed.